operator/v1/types_network.go

```diff
@@ -507,6 +507,20 @@ type GatewayConfig struct {
 	// The supported values are "Restricted" and "Global".
 	// +optional
 	IPForwarding IPForwardingMode `json:"ipForwarding,omitempty"`
+	// V4InternalMasqueradeSubnet contains the v4 masquerade addresses used internally by
+	// ovn-kubernetes to enable host to service traffic. The host is configured with these
+	// addresses, as well as the shared gateway bridge interface.The values can be changed after
+	// installation.
+	// The default subnet is 169.254.169.0/29
```

Comment:
I think there's a regex that can validate this.  I bet @Miciah knows for this and for the ipv6 one.

Reply:
we dont currently use regex to validate the other configurable subnet in ovn-k (the internal join subnet), and any invalid value passed in would be caught by the cno or in ovnk, do you think we need that here?

Reply:
added regex taken from https://github.com/openshift/api/blob/3e3f07aadec46cad1812b122a51ce83559934054/operator/v1/types_ingress.go#L378-L386

Reply:
split into v4 and v6 respectively